frontend/integration-tests/tests/kubevirt/vm.overview.scenario.ts

```diff
@@ -123,4 +122,67 @@ describe('Test vm overview page', () => {
       });
     });
   });
+
+  describe('Test vm overview on project overview page', () => {
+    const newVMDescription = 'edited vm description';
```

Comment:
Duplicated. Please, define this on one place only.

Reply:
yeah, I've been thinking on this for a while, will change it.